apps/hannk/interpreter/stubs.cpp

```diff
@@ -0,0 +1,19 @@
+#include "stdio.h"
```

Comment:
Please run clang-format on this.

Reply:
Also please add comments about what it's for, etc.

Reply:
Done